### PR TITLE
Create SpecialPowers + EventUtils + QUnit testing template

### PR DESCRIPTION
**Pull request by humphd**

### CODE REVIEW

File: test/ui/default.conf [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
```
Comment (jbuck) [inline test/ui/default.conf]: Depending on whether lighthouse 1082 lands before this or not, we should nuke this new config file

File: test/ui/test.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+         **/
+        test( "window.SpecialPowers", function(){
+          ok( "SpecialPowers" in window, "SpecialPowers extension is installed." );
+        });
+
+      }
+
+      media.onReady( start );
+
+      window.butter = butter;
```
Comment (jbuck) [inline test/ui/test.js]: delete this

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_cleanup** from @jbuck received. Current queue size: 0

Live output at: http://botio.butyr.org:80/9a63710b6a7a219/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:80/9a63710b6a7a219/output.txt

Total script time: 0.13 mins

**Pull request merged**